LayoutTests/crypto/workers/reportingObserverNonNull.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
+<script src="../resources/common.js"></script>
+</head>
+<body>
+<p id="description"></p>
+<div id="console"></div>
+<script>
+  description("Test generating a ReportingObserver. PASSING if not crashing");
```

Comment:
I would have expected the description string to show up in -expected.txt.
The fact there is a syntax error seems like an issue.